security/integration/utils/kubernetes.go

```diff
@@ -135,30 +141,32 @@ func CreatePod(clientset kubernetes.Interface, namespace string, image string, n
 	pod := &v1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
 			Labels: map[string]string{
-				"uuid":      uuid,
+				"uuid":      poduuid,
 				"pod-group": fmt.Sprintf("%v-pod-group", name),
 			},
 			Name: name,
 		},
 		Spec: spec,
 	}
 
-	pod, err := clientset.CoreV1().Pods(namespace).Create(pod)
+	_, err := clientset.CoreV1().Pods(namespace).Create(pod)
 	if err != nil {
 		return nil, err
 	}
 
-	if err := waitForPodRunning(clientset, namespace, uuid, 60*time.Second); err != nil {
+	if err := waitForPodRunning(clientset, namespace, poduuid, 60*time.Second); err != nil {
 		return nil, err
 	}
 
 	return clientset.CoreV1().Pods(namespace).Get(name, metav1.GetOptions{})
 }
 
+// DeletePod takes ideneity information of a pod and deletes the pod.
 func DeletePod(clientset kubernetes.Interface, namespace string, name string) error {
 	return clientset.CoreV1().Pods(namespace).Delete(name, &metav1.DeleteOptions{})
 }
 
+// CreateRole creates a role object named "istio-ca-role".
 func CreateRole(clientset kubernetes.Interface, namespace string) error {
```

Comment:
Rename this function to "CreateIstioCARole" to be precise?

Reply:
Done. Thanks!